### PR TITLE
MGMT-7178: Assisted-installer should not reboot worker till min number of masters have joined

### PR DESCRIPTION
**Pull request by ori-amizur**
/cc @tsorya

### CODE REVIEW

File: src/installer/installer.go [hunk 1]
```diff
@@ -133,6 +133,11 @@ func (i *installer) InstallNode() error {
 		}
 		i.log.Info("Setting bootstrap node new role to master")
 
+	} else if i.Config.Role == string(models.HostRoleWorker) {
```
Comment (tsorya) [inline src/installer/installer.go:136]: why to wait for controller? i think you should just wait till 2 masters (not bootstrap) moved to Joined or Done

Comment (ori-amizur) [reply src/installer/installer.go:136]: OK

Comment (tsorya) [inline src/installer/installer.go:136]: i am not sure 100% but there is an option it will break day2 flow, as there will be none masters in the cluster

**Comment by flaper87** [comment]
/retitle MGMT-7178: Assisted-installer should not reboot worker till min number of masters have joined

**Review by flaper87 — APPROVED** [review]
Please, always add a note when holding a PR

Why is this PR on hold?

**Comment by ori-amizur** [comment]
> Please, always add a note when holding a PR
> 
> Why is this PR on hold?

Waited for https://github.com/openshift/assisted-service/pull/2399

**Pull request merged**